app/ytdl.py

```diff
@@ -234,7 +234,7 @@ async def __add_entry(self, entry, quality, format, folder, already):
             if not self.queue.exists(entry['id']):
                 dl = DownloadInfo(entry['id'], entry['title'], entry.get('webpage_url') or entry['url'], quality, format, folder)
                 # Keep consistent with frontend
-                base_directory = self.config.DOWNLOAD_DIR if (quality != 'audio' and format != 'mp3') else self.config.AUDIO_DOWNLOAD_DIR

```

Comment:
I know this code has already been here before, but I'm wondering if the condition makes sense -- why predicate on the format? It makes sense to me that if `quality == "audio"` we should use `AUDIO_DOWNLOAD_DIR`, and that's that

Reply:
Looks like a writing habit problem.  When quality is not audio and format is not mp3, do nothing, otherwise use audio dir.  
That is, when the quality is audio or the format is mp3, use audio dir(⁠ノ⁠｀⁠Д⁠´⁠)⁠ノ⁠彡⁠┻⁠━⁠┻

Reply:
actually I do see the logic. we can select audio both in the quality and in the format. it's a bit of a mess. oh well.

Reply:
Yes that is why I was intended in the beginning to drop Any that would needed to have Audio selected as quality.